src/translations/v6/v6.sbvr

```diff
@@ -187,6 +187,9 @@ Term: memory usage
 Term: memory total
 	Concept Type: Integer (Type)
 
+Term= secureboot keys metadata
```

Comment:
Personally I wouldn't expose this on v6, if it's info people want then they can update to using v7